ui/admin/app/components/form/target/add-host-sets/index.hbs

```diff
@@ -17,37 +17,33 @@
       @cancelText={{t 'actions.cancel'}}
     />
 
-    <Rose::Table as |table|>
-      <table.header as |header|>
-        <header.row as |row|>
-          <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-          <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-          <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
-          <row.headerCell>{{t 'resources.host-catalog.title'}}</row.headerCell>
-        </header.row>
-      </table.header>
-      <table.body as |body|>
-        {{#each this.filteredHostSets as |hostSet|}}
-          <body.row as |row|>
-            <row.cell>
-              <form.checkbox
-                @label={{hostSet.id}}
-                @description={{hostSet.description}}
-                @onChange={{fn this.toggleHostSet hostSet}}
-              />
-            </row.cell>
-            <row.cell>{{hostSet.name}}</row.cell>
-            <row.cell>
-              <HostCatalogTypeBadge @model={{hostSet}} />
-            </row.cell>
-            <row.cell>
-              <code>{{hostSet.host_catalog_id}}</code>
-            </row.cell>
-          </body.row>
-        {{/each}}
-      </table.body>
-    </Rose::Table>
-
+    <Hds::Table
+      @model={{this.filteredHostSets}}
+      @columns={{array
+        (hash label=(t 'form.id.label'))
+        (hash label=(t 'form.name.label'))
+        (hash label=(t 'form.type.label'))
+        (hash label=(t 'resources.host-catalog.title'))
+      }}
+      @valign='middle'
+    >
+      <:body as |B|>
+        <B.Tr>
+          <B.Td>
+            <Hds::Form::Checkbox::Field
+              {{on 'change' (fn this.toggleHostSet B.data.id)}}
+              as |F|
+            >
+              <F.Label>{{B.data.id}}</F.Label>
+              <F.HelperText>{{B.data.description}}</F.HelperText>
+            </Hds::Form::Checkbox::Field>
+          </B.Td>
+          <B.Td>{{B.data.name}}</B.Td>
+          <B.Td><HostCatalogTypeBadge @model={{B.data}} /></B.Td>
+          <B.Td><code>{{B.data.host_catalog_id}}</code></B.Td>
```

Comment:
Should the `host_catalog_id` be embedded in a `<Copyable>` component ?

Reply:
This route is for adding host sources to a target. Not sure what the user would do with the copied host catalog id from this view, so I don't think it is necessary to wrap it with copyable. thoughts?

Reply:
Good point, ill leave it as is then. Thank you!